pom.xml

```diff
@@ -224,7 +231,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                	<artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <argLine>-Xms512m -Xmx2048m -XX:MaxPermSize=768m -XX:ReservedCodeCacheSize=128m</argLine>
+                    <systemPropertyVariables>
+                        <org.jboss.logging.provider>jdk</org.jboss.logging.provider>
```

Comment:
Without this patch applied, I see that line about 40 times. With that patch applied, I see that line about 40 times. I also do not understand how this property could accect that line?